### DISCUSSION TITLE
How can I make Qtile change monitor focus with a dual monitor setup

### DISCUSSION THREAD

**Discussion by hiroi-p** [qtile/qtile#2637] [Q&A]
Hi! So I have a 2 monitor setup (main monitor is gonna be A and the second one B) and I want qtile to change the focus of the monitor when I move my cursor from screen A to B. I changes fine if there's a window open on screen B but when there's nothing the focus remains on the initial screen while I try to work change workspaces on the other. 

The only way to change it is by clicking the monitor that has nothing on it, but it's a but tedious and I want to remove the unnecessary click and to make it feel flawless. 

This is my config file:  https://pastebin.com/j0JbAiwL

Thank you, I really want to make it work.

**Comment by botanicEventuality** [comment]
Try adding this keybinding:
`Key([mod], 'period', lazy.next_screen(), desc='Next monitor'),`
I mean, that's the one I use, but just add any keybinding you want to `lazy.next_screen()`
Hope it helps!

**Reply by yobleck** [reply]
See issue #2657

**Reply by LumpTatorFace** [reply]
Thanks man, worked like a treat :)

**Comment by SrShelo** [comment]
I recommend establishing cursor_warp = True. That will cause the cursor to teleport to the focused screen.
